Automattic/pocket-casts-android
# Playback 2023 - Listened numbers story

| 📘 Part of: https://github.com/Automattic/pocket-casts-android/issues/1463 | 🎨 Designs: lH66LwxxgG8btQ8NrM0ldx-fi-1599_20385 |
|:---:|:---:|

Adds the 2023 visual for the Listened Numbers Story and updates the order so it's the first story (after the intro).

**Animations are not part of this task and will be tackled later.**

## To test

1. Make sure you're logged in into an account that has a few episodes listened
2. Go to Profile tab
3. Tap the EOY image
4. Skip to the second story
5. ✅ Check that the Listened Numbers story looks good
6. Tap "Share this story" 
7. Share the story
8. ✅ The story image should look good with a proper blur effect


## Screenshots or Screencast 

Pixel 7 (normal display) | Pixel 7 (large display) | Pixel 5 (API 24) 
----|----|----
![image](https://github.com/Automattic/pocket-casts-android/assets/1405144/36e803bc-60b0-430d-816f-ff4ab27fa8b8) | ![image](https://github.com/Automattic/pocket-casts-android/assets/1405144/5bbb6493-e1c8-4950-8de9-6c41750dd081) | ![image](https://github.com/Automattic/pocket-casts-android/assets/1405144/530d6652-c1ee-443c-b6c6-d9e81c761574) | 


| Nexus 9 Tablet | 
|----|
|<img  width=400 src="https://github.com/Automattic/pocket-casts-android/assets/1405144/5fd28d91-19eb-4578-b1b9-8d150a55b313"/>|

| Shared Story Image| 
|----|
|<img  width=300 src="https://github.com/Automattic/pocket-casts-android/assets/1405144/8435f6d1-e769-447a-8d78-76a8347ef4aa"/>|


## Checklist
 
#### I have tested any UI changes...
